.github/dependabot.yml

```
@@ -5,7 +5,37 @@
 
 version: 2
 updates:
-  - package-ecosystem: "" # See documentation for possible values
-    directory: "/" # Location of package manifests
+  # Gradle - Root project
+  - package-ecosystem: "gradle"
+    directory: "/"
     schedule:
       interval: "daily"
+    open-pull-requests-limit: 10
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)